tests/phpunit/tests/KSES.php

```diff
@@ -308,6 +332,32 @@ public function test_filter_kses_allowed_html( $html, $expected ) {
 		remove_filter( 'wp_kses_allowed_html', [ $kses, 'filter_kses_allowed_html' ] );
 	}
 
+	/**
+	 * Testing the filter_kses_allowed_html() method.
+	 *
+	 * @covers ::filter_kses_allowed_html
+	 * @covers ::array_merge_recursive_distinct
+	 */
+	public function test_filter_kses_allowed_html_uses_deep_merge() {
+		$kses         = new \Google\Web_Stories\KSES();
+		$allowed_tags = [
+			'img' => [
+				'width' => true,
```

Comment:
PHPCS warning

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6542/commits/055bdbcac14e27cd8b382d8e5cd6c510262af304